products/rhel8/profiles/stig.profile

```diff
@@ -443,6 +443,9 @@ selections:
     # RHEL-08-010730
     - file_permissions_home_directories
 
+    # RHEL-08-010731
+    - accounts_users_home_files_permissions
```

Comment:
When you add a rule to a RHEL 8 profile you need to assign a CCE to the rule. Pick a random number from the shared/references/cce-redhat-avail.txt, remove it from there and create a `cce@rhel8:` key in the rule.yml in the identifiers section and put it there.